network/init-neutrond.sh

```diff
@@ -53,6 +53,12 @@ CW4_GROUP_CONTRACT=$THIRD_PARTY_CONTRACTS_DIR/cw4_group.wasm
 
 NEUTRON_CHAIN_MANAGER_CONTRACT=$CONTRACTS_BINARIES_DIR/neutron_chain_manager.wasm
 
+# Slinky genesis configs
+USE_CORE_MARKETS=true
```

Comment:
may be we can set those as env variables with a default values? For easy configuration without editing the script
`USE_CORE_MARKETS=${USE_CORE_MARKETS:-true}